modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/InstanceListPageItem.es.js

```diff
@@ -52,8 +52,10 @@ function Item({totalCount, ...instance}) {
 	const assignedToUser = !!assignees.find(({id}) => id === Number(userId));
 	const assigneeNames = assignees.map((user) => user.name).join(', ');
 	const {reviewer} = assignees.find(({id}) => id === -1) || {};
+	const hasTasksWithoutAssignee = !!assignees.find(({id}) => id === -1);
```

Comment:
Hey @seltong, the variable `hasTasksWithoutAssignee` is not required. You are able to achieve the same result using the `reviewer`.

Thx